assets/src/edit-story/components/canvas/singleSelectionMovable.js

```diff
@@ -245,6 +250,21 @@ function SingleSelectionMovable({ selectedElement, targetEl, pushEvent }) {
         if (updates && updates.height) {
           newHeight = dataToEditorY(updates.height);
         }
+
+        if (isResizingFromCorner) {
+          if (newWidth < minWidth) {
+            newWidth = minWidth;
+            newHeight = newWidth / aspectRatio;
```

Comment:
I think the `aspectRatio` should be taken not from the initial `selectedElement`, but from the `width` and `height` passed here in the arguments. That would also make this code easier to apply for the multi-selection.

Reply:
I started with that, unfortunately, when resizing from the corner we don't get consistent values (rounding?) and it leads to very weird results when hitting min size (element can grow unexpectedly).

[Video with inconsistent aspect ratio](https://drive.google.com/open?id=1sB52bI2-HPNXMF1NuXN1DxvSw4cCaFga).

It is also affecting the final width/height so maybe we should try to fix that? On video: `200x200->401x400` I think we can fix that 401.